src/db/tables/subscriber_churns.ts

```diff
@@ -59,6 +59,29 @@ async function getAllSubscriberChurns(): Promise<SubscriberChurnRow[]> {
   }
 }
 
+async function getUpcomingChurns(): Promise<Array<SubscriberChurnRow>> {
```

Comment:
This is the same as the `getChurnsToEmail` query, with the join and `churn_prevention_email_sent_at` check removed.